src/govuk/components/accordion/accordion.mjs

```diff
@@ -31,13 +31,11 @@ var TRANSLATIONS_DEFAULT = {
  *
  * @class
  * @param {HTMLElement} $module - HTML element to use for accordion
- * @param {object} config - Accordion config
- * @param {AccordionTranslations} config.i18n - Translations
+ * @param {AccordionTranslations} [config] - Accordion config
```

Comment:
```suggestion
 * @param {AccordionConfig} [config] - Accordion config
```

Reply:
Brilliant spot. All fixed and ready for you again 😊